src/System.IO/src/System/IO/StreamReader.cs

```diff
@@ -962,8 +997,11 @@ public override Task<int> ReadAsync(char[] buffer, int index, int count)
 
             return task;
         }
-
+#if uap101aot
```

Comment:
Is there an issue# associated with this?  Could you include a TODO with it for this to be removed?